v2/googlecloud-to-elasticsearch/docs/GCSToElasticsearch/README.md

```diff
@@ -248,3 +250,9 @@ gcloud beta dataflow flex-template run ${JOB_NAME} \
         --template-file-gcs-location=${TEMPLATE_IMAGE_SPEC} \
         --parameters inputFileSpec=${INPUT_FILE_SPEC},connectionUrl=${CONNECTION_URL},index=${INDEX},elasticsearchUsername=${ELASTICSEARCH_USERNAME},elasticsearchPassword=${ELASTICSEARCH_PASSWORD},containsHeaders=${HEADERS},deadletterTable=${DEADLETTER_TABLE},delimiter=${DELIMITER}
```

Comment:
Instead of elasticsearchUsername & elasticsearchPassword better to add apiKey as an option